docs/source/transforms.rst

```diff
@@ -171,6 +171,12 @@ Transforms are common audio transforms. They can be chained together using :clas
 
   .. automethod:: forward
 
+:hidden:`RNNTLoss`
```

Comment:
This makes me wonder if transforms should have subsections similar to functionals doc.

Reply:
How about creating a loss category? We have Si-SDR and RNNTLoss so far, maybe we could add more under this subsection.

Reply:
I've created a new `Loss` section in [functional docs](https://304212-90321822-gh.circle-artifacts.com/0/docs/functional.html) for `rnnt_loss` in this PR, but the [transforms docs](https://304212-90321822-gh.circle-artifacts.com/0/docs/transforms.html) is lacking these similar subsections and instead treats each transform independently. I think it makes sense to add subsections for transforms, especially since the number of transforms is growing, and we can follow up on this in a separate PR.

Reply:
Yes, we can add subsection in transforms doc in a separate PR.

Reply:
> How about creating a loss category? We have Si-SDR and RNNTLoss so far, maybe we could add more under this subsection.

I am not confident with the implementation so, if someone ports it, we need to double-check the implementation.